website/blog/2023-05-07-1M-milestone/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Surpassing 1 Million Downloads - A Retrospective and a Look into the Future
+authors: qingyunwu
+tags: [LLM, LLMOps, FLAMLv2]
+---
+
+**TL;DR:**
+* **Celebrating FLAML's milestone: 1 million downloads**
+* **Introducing Large Language Model (LLM) support in the upcoming FLAML v2**
+
+
+This week, FLAML has reached a significant milestone: 1 million downloads. Originating as an intern research project within Microsoft Research, FLAML has grown into an open-source library used widely across the industry and supported by an active community.
+As we celebrate this milestone, we want to recognize the passionate contributors and users who have played an essential role in molding FLAML into the flourishing project it is today. Our heartfelt gratitude goes out to each of you for your unwavering support, constructive feedback, and innovative contributions that have driven FLAML to new heights.
+
+We'd also like to take the opportunity to reflect on FLAML's past achievements and its future roadmap, with a particular focus on large language models (LLM) and LLMOps.
+
+## FLAML's Journey: Past Achievements and Milestones
+
+### Bring AutoML to One's Fingertips
+FLAML offers an off-the-shelf AutoML solution that enables users to quickly discover high-quality models or configurations for common ML/AI tasks. By automatically selecting models and hyperparameters for training or inference, FLAML saves users time and effort. FLAML has significantly reduced development time for developers and data scientists alike, while also providing a convenient way to integrate new algorithms into the pipeline, enabling easy extensions and large-scale parallel tuning. These features make FLAML a valuable tool in R&D efforts for many enterprise users.
+
+### Speed and Efficiency: The FLAML Advantage
+What sets FLAML apart from other AutoML libraries is its efficiency. Both [`flaml.tune`](https://microsoft.github.io/FLAML/docs/Use-Cases/Tune-User-Defined-Function) and [flaml.AutoML](https://microsoft.github.io/FLAML/docs/Use-Cases/Task-Oriented-AutoML) modules are fast, thanks to efficient hyperparameter optimization and model selection methods. FLAML is also capable of handling large search spaces with heterogeneous evaluation costs, complex constraints, guidance, and early stopping.
```

Comment:
In Python, a module refers to a .py file. So I feel odd when seeing "modules".

Reply:
Removed the word "modules". Thanks!